src/alfasim_sdk/_internal/hook_specs_gui.py

```diff
@@ -173,10 +173,10 @@ def alfasim_get_status(
 
     The status monitor accepts two types of message:
 
-     - :func:`~alfasim_sdk.status.ErrorMessage`:
+    :class:`alfasim_sdk.ErrorMessage`:
```

Comment:
If possible (I am not sure if the behavior is the same in `:class:`), add the `~` in front of the `alfasim_sdk.ErrorMessage`.